### PR TITLE
K8s: release note updates

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Remove TGK 2.1/2.2 and add RED-106834 limitation.

### CODE REVIEW

File: content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md [hunk 1]
```diff
@@ -116,12 +114,15 @@ Any distribution not listed below is not supported for production workloads.
 
 The limitations below are specific to the Redis Enterprise for Kubernetes product. To view limitations for the supported Redis Enterprise Software version, see the [7.2.4 release notes]({{<relref "/rs/release-notes/rs-7-2-4-releases#known-limitations">}}).
 
-{{<warning>}}In version 7.2.4, old module versions and manually uploaded modules are not persisted. If databases are not upgraded after cluster upgrade, and require cluster recovery afterwards, you'll need to contact Redis support. This issue will be fixed in the next maintenance release by moving the stored location of the modules.{{</warning>}}
 
 ### New limitations
 
+- **REDBs created with version 6.2.10-4 or before will automatically upgrade modules when the cluster is upgraded (RED-106834)**
```
Comment (alon-zada) [inline content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md:120]: only before th 6.2.10-4 was fixed.

File: content/kubernetes/release-notes/7-2-4-releases/_index.md [hunk 2]
```diff
@@ -83,6 +81,8 @@ The limitations below are specific to the Redis Enterprise for Kubernetes produc
 
 - **Cannot upgrade or recover clusters if there are databases with manually uploaded modules (RED-107711 RED-106872)** Contact support.
 
+- **REDBs created with version 6.2.10-4 or before will automatically upgrade modules when the cluster is upgraded (RED-106834)**
```
Comment (alon-zada) [inline content/kubernetes/release-notes/7-2-4-releases/_index.md:84]: same here as above.

File: content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md [hunk 3]
```diff
@@ -116,12 +114,15 @@ Any distribution not listed below is not supported for production workloads.
...
+
 - **Cannot upgrade or recover clusters if there are databases with manually uploaded modules (RED-107711 RED-106872)** Contact support.
 
+{{<warning>}}In version 7.2.4, old module versions and manually uploaded modules are not persisted. If databases are not upgraded after cluster upgrade, and require cluster recovery afterwards, you'll need to contact Redis support. This issue will be fixed in the next maintenance release by moving the stored location of the modules.{{</warning>}}
```
Comment (alon-zada) [inline content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md:124]: I think we should not promise that this will be fixed in the next maint release.

**Review by alon-zada — CHANGES_REQUESTED** [review]
Great!
Please view inline comments.

File: content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md (hunk 3 above)
Comment (bascar) [reply content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md:124]: I don't have a problem with this. Already fixed in RS and CP'd into the Sep 13 RS maintenance release, as planned.

**Review by bascar — CHANGES_REQUESTED** [review]
LGTM, plz address the version notes from Alon.

File: content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md (hunk 3 above)
Comment (alon-zada) [reply content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md:124]: ok, but keep in mind that wasn't tested via us on K8s only in theory (though I don't think this should not work).

File: content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md (hunk 1 above)
Comment (kaitlynmichael) [inline content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md:120]:
```suggestion
- **REDBs created before version 6.2.10-4 will automatically upgrade modules when the cluster is upgraded (RED-106834)**
```

**Review by rrelledge — APPROVED** [review]

**Pull request merged**